peps/pep-0773.rst

```diff
@@ -1099,6 +1099,13 @@ almost equivalent packages from NuGet (the latter does not include Tcl/Tk,
 making them significantly incompatible for some users, but this is likely okay
 for especially old versions).
 
+As of this PEP's acceptance, the plan is to create packages for all versions of
+Python 3.10.0 onwards (excluding prereleases), so that all non-EOL releases can
```

Comment:
I assume this captures the intent, I thought it might be useful to be explicit. Will need updating for line length if accepted.
```suggestion
Python 3.10.0 onwards (including patch releases, but excluding pre-releases), so that all non-EOL releases can
```